_data/sites.json

```
@@ -17219,6 +17275,18 @@
         ]
     },
 
+    {
+        "name": "Pearson",
+        "url": "https://www.pearson.com/privacy-center/privacy-notices/full-privacy-notice.html#12",
+        "difficulty": "limited",
+        "email": "dataprivacy@pearson.com",
+        "notes": "For general questions and requests you can email. Information for how to make a CCPA request is located here: 'https://www.pearson.com/en-us/legal-information/california-consumer-rights.html'",
```

```suggestion
        "notes": "For general questions and requests you can email. Information for how to make a CCPA request is located [here](https://www.pearson.com/en-us/legal-information/california-consumer-rights.html)'",
```
Use markdown link formatting so it looks better on the website